src/Laravel/Eloquent/Metadata/ModelMetadata.php

```diff
@@ -65,8 +66,14 @@ public function getAttributes(Model $model): Collection
         $connection = $model->getConnection();
         $schema = $connection->getSchemaBuilder();
         $table = $model->getTable();
-        $columns = $schema->getColumns($table);
-        $indexes = $schema->getIndexes($table);
+        /** @var array<string, mixed> $columns */
+        $columns = Cache::flexible('api-platform.tables.'.$table, [5, 10], function () use ($schema, $table) {
+            return $schema->getColumns($table);
+        });
+        /** @var array<int, mixed> $indexes */
+        $indexes = Cache::flexible('api-platform.indexes.'.$table, [5, 10], function () use ($schema, $table) {
+            return $schema->getIndexes($table);
+        });
```

Comment:
Interesting change, note also https://github.com/api-platform/core/issues/6735

Reply:
@soyuka I logged all the queries and saw multiple calls to the information_schema table for the same table's columns and data and thats why I added this

Reply:
@soyuka in terms of making sure it is not stale:
https://laravel.com/docs/11.x/cache#swr

But as you pointed out, in production this is not called because it cached when calling the `optimize` function (or after the 1st request to the endpoint), so this would apply only to development machines, im not quite sure if or why it would be needed there.

@amermchaudhary Are you running into this issue on a production setup or on your development system?

Reply:
yes @amermchaudhary it's running multiple time on a development environment, but if you set debug to `false` metadata gets cached. I'm not a huge fan of multi-layer caching as it's hard to debug when it fails.

Reply:
Ahh I was checking on my development system. I didn't know that it was being cached on a production system. In that case we should close this PR. It's not necessary!!